### PR TITLE
adds data source page

### PR DESCRIPTION
**Pull request by J05h-L**
### Context
[AB#212292](https://dfe-ssp.visualstudio.com/a14e55df-4fbf-4a2f-a11d-22b187178343/_workitems/edit/212292) - [AB#213457](https://dfe-ssp.visualstudio.com/a14e55df-4fbf-4a2f-a11d-22b187178343/_workitems/edit/213457)

### Change proposed in this pull request
Adds content for the data source page.
Allows download of the AAR and CFR data for the last 9 years from storage as excel files from this page.
adds a link for this page to the resources navigation section from school, trust and la home pages.
adds integration tests.

### Guidance to review 
N/A

### Checklist
- [x] Work items have been linked [(use AB#)](https://learn.microsoft.com/en-us/azure/devops/boards/github/link-to-from-github?view=azure-devops#use-ab-to-link-from-github-to-azure-boards-work-items)
- [x] Your code builds clean without any errors or warnings
- [x] You have run all unit/integration tests and they pass
- [x] Your branch has been rebased onto main
- [x] You have tested by running locally

### CODE REVIEW

File: web/src/Web.App/Views/DataSource/_DataSources.cshtml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+﻿<h2 class="govuk-heading-m">Data Sources</h2>
+
+<h3 class="govuk-heading-s">Which types of schools can I get data for?</h3>
+
+<p class="govuk-body">
+    The data on the site is for maintained schools and academies in England. It’s based on schools’ Consistent Financial Reporting (CFR) returns and academies’ accounting returns.
```
Comment (WolfyUK) [inline web/src/Web.App/Views/DataSource/_DataSources.cshtml:6]: chore: Replace `’` with `'`

File: web/src/Web.App/Views/DataSource/_DataSources.cshtml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    You can also get data for multi-academy trusts (MATs).
+</p>
+
+<h3 class="govuk-heading-s">Financial returns</h3>
+
+<p class="govuk-body">
+    This is based on schools’ Consistent Financial Reporting (CFR) returns for LA maintained schools and academies’ accounting returns (AARs).
```
Comment (WolfyUK) [inline web/src/Web.App/Views/DataSource/_DataSources.cshtml:13]: chore: Replace `’` with `'`

File: web/src/Web.App/Views/DataSource/_Interpretation.cshtml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+﻿<h2 class="govuk-heading-m">Interpreting the data</h2>
+
+<h3 class="govuk-heading-s">Interpreting expenditure charts</h3>
+
+<p class="govuk-body">
+    You should take care when interpreting the data. It shows the position of a school relative to others; it doesn’t explain why this is.
```
Comment (WolfyUK) [inline web/src/Web.App/Views/DataSource/_Interpretation.cshtml:6]: chore: Replace `’` with `'`

File: web/src/Web.App/Views/DataSource/_TeachingCosts.cshtml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+﻿<h4 class="govuk-heading-s">Teaching and teaching support staff costs</h4>
+
+<p class="govuk-body">
+    As staffing typically represents between 75% and 80% of mainstream schools’ expenditure, and more for special schools and pupil referral units (PRUs),
```
Comment (WolfyUK) [inline web/src/Web.App/Views/DataSource/_TeachingCosts.cshtml:4]: chore: Replace `’` with `'`

File: web/src/Web.App/Views/DataSource/_TeachingCosts.cshtml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    it’s important that schools review their staff structures regularly as part of their annual school improvement, curriculum and financial planning.
+</p>
+
+<p class="govuk-body">
+    When you are interrogating other schools’ finances to compare with your own, it’s important to look at the education delivery model as a whole.
+    The data that sits within the workforce tab on the benchmark charts page will help with this.
+</p>
+
+<p class="govuk-body">
+    It is also useful to have discussions with professionals in other schools in your benchmark set to grain greater insight into the reasons
```
Comment (WolfyUK) [inline web/src/Web.App/Views/DataSource/_TeachingCosts.cshtml:14]: chore: Typo in `grain`

File: web/src/Web.App/Views/DataSource/_DataSources.cshtml [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+</p>
+
+<h4 class="govuk-heading-s">LA Maintained Schools</h4>
+
+<ul class="govuk-list">
+    @foreach (var dataSourceFile in Model.MaintainedSchools)
+    {
+        <li>
+            <a target="_blank" rel="external noopener noreferrer" class="govuk-link" href=@dataSourceFile.Link>
```
Comment (WolfyUK) [inline web/src/Web.App/Views/DataSource/_DataSources.cshtml:22]: suggestion: This link will reveal the location in blob storage, including the SAS Token. An alternative consideration could be an API action that proxies this request instead

Comment (jrabbott) [reply web/src/Web.App/Views/DataSource/_DataSources.cshtml:22]: I'm happy with this approach. 

It's a separate storage account, and the sas is time bound & limited access.

**Review by jrabbott — APPROVED** [review]

**Pull request merged**